release-doc-in-process/all_vf.TTL

```diff
@@ -179,33 +153,12 @@ vf:committedOutputs
         rdfs:domain         vf:Process ;
         rdfs:range          vf:Commitment .
 
-vf:resourceClassifiedAs
-        a                   owl:ObjectProperty ;
-        rdfs:label          "resource classified as" ;
-        rdfs:domain         [ owl:unionOf (vf:Commitment vf:EconomicResource) ] ;
-        rdfs:range          vf:ResourceClassification ;
-        rdfs:comment        "A resource's primary classification, as part of describing that resource." .
-
-vf:processClassifiedAs
-        a                   owl:ObjectProperty ;
-        rdfs:label          "process classified as" ;
-        rdfs:domain         vf:Process ;
-        rdfs:range          vf:ProcessClassification ;
-        rdfs:comment        "The primary classification that describes a process, at any level of granularity." .
-
-vf:exchangeClassifiedAs
-        a                   owl:ObjectProperty ;
-        rdfs:label          "exchange classified as" ;
-        rdfs:domain         vf:ExchangeAgreement ;
-        rdfs:range          vf:ExchangeClassification ;
-        rdfs:comment        "The primary classification that describes an exchange agreement, at any level of granularity." .
-
-vf:transferClassifiedAs
+vf:classifiedAs
         a                   owl:ObjectProperty ;
-        rdfs:label          "exchange classified as" ;
-        rdfs:domain         vf:Transfer ;
-        rdfs:range          vf:TransferClassification ;
-        rdfs:comment        "The primary classification that describes a transfer, at any level of granularity." .
+        rdfs:label          "classified as" ;
+        rdfs:domain         owl:Thing ;
+        rdfs:range          owl:Thing ;
+        rdfs:comment        "References concept in a common taxony for purpose of categorizing." .
```

Comment:
Suggestion on definition:  References a concept in a common taxonomy or other classification scheme for purposes of categorization.